File: week-07/day-2/Workshop 16.5.2023/append_a_func.py
# - Create a variable named `typo` and assign the value `Chinchill` to it
# - Write a function called `append_a()` that gets a string as an input,
#   appends an 'a' character to its end and returns with a string
# - Print the result of `append_a(typo)`

typo = 'Chinchill'


def append_a(string):
    my_string = "a"
    return string+my_string


print(append_a(typo))

File: week-02/day-3/2023.04.12/src/AppendAFunc.java


public class AppendAFunc {
    public static void main(String[] args) {

        String typo = "Chinchill";

        appendA(typo);


    }

    public static void appendA (String myString) {
       myString = myString.concat("a");
        System.out.println(myString);
    }
}
